langchain4j-mcp/src/test/java/dev/langchain4j/mcp/client/ToolSpecificationHelperTest.java

```diff
@@ -0,0 +1,189 @@
+import dev.langchain4j.model.chat.request.json.JsonArraySchema;
+import dev.langchain4j.model.chat.request.json.JsonBooleanSchema;
+import dev.langchain4j.model.chat.request.json.JsonEnumSchema;
+import dev.langchain4j.model.chat.request.json.JsonIntegerSchema;
+import dev.langchain4j.model.chat.request.json.JsonNumberSchema;
+import dev.langchain4j.model.chat.request.json.JsonObjectSchema;
+import dev.langchain4j.model.chat.request.json.JsonStringSchema;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class ToolSpecificationHelperTest {
+
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @Test
+    public void toolWithSimpleParams() throws JsonProcessingException {
+        String text =
+                """
+                [ {
+                      "name" : "operation",
+                      "description" : "Super operation",
+                      "inputSchema" : {
+                        "type" : "object",
+                        "properties" : {
+                          "stringParameter" : {
+                            "type" : "string",
+                            "description" : "Message to echo"
+                          },
+                          "enumParameter" : {
+                            "type" : "string",
+                            "description" : "The protocol to use",
+                            "enum": [
+                                "http",
+                                "https"
+                            ]
+                          },
+                          "numberParameter": {
+                            "type": "number",
+                            "description": "A number"
+                          },
+                          "integerParameter": {
+                            "type": "integer",
+                            "description": "An integer"
+                          },
+                          "booleanParameter": {
+                            "type": "boolean",
+                            "description": "A boolean"
+                          },
+                          "arrayParameter": {
+                              "type": "array",
+                              "description": "An array of strings",
+                              "items": {
+                                "type": "string"
+                              }
+                          }
+                        },
+                        "required" : [ "stringParameter" ],
+                        "additionalProperties" : false,
+                        "$schema" : "http://json-schema.org/draft-07/schema#"
+                      }
+                    } ]
+                """;
+        ArrayNode json = OBJECT_MAPPER.readValue(text, ArrayNode.class);
+        List<ToolSpecification> toolSpecifications = ToolSpecificationHelper.toolSpecificationListFromMcpResponse(json);
+        assertEquals(1, toolSpecifications.size());
```

Comment:
nit: IMO assertJ assertions are more expressive

Reply:
Ok, I rewrote them (I generally prefer pure JUnit, but it may just be a force of habit..)